aten/src/ATen/SparseGCSTensorImpl.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <ATen/Tensor.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/SparseTensorUtils.h>
+#include <ATen/SparseCOOTensorImpl.h>
+#include <c10/core/TensorImpl.h>
+#include <c10/util/Exception.h>
+
+namespace at {
+  
+// TODO: since many methods in SparseCOOTensorImpl can be used by GCS sparse tensor directly
+// we probably should have some superclass between TensorImpl and GCSTensorImpl that is
+// shared between COO and GCS tensors.
```

Comment:
I don't think this is actually true

Reply:
Yeah this comment is pretty old, before the two started diverging too much. Will remove.